src/site/includes/debug.drupal.liquid

```diff
@@ -1,4 +1,4 @@
-{% if buildtype != 'vagovprod' %}
+{% if buildtype != 'vagovprod' and omitdebug == false %}
```

Comment:
I'm not seeing the debug data on staging template URLs anymore, could this have broken it by chance? @timwright12 e.g. http://preview-staging.vfs.va.gov/preview?nodeId=16139